integration_tests/models/source/snowflake/snowplow_mobile_events_stg.sql

```diff
@@ -417,7 +426,8 @@ select
     event_fingerprint,
     true_tstamp,
     object_construct('id',id,'name',name,'previousId',previousId,'previousName',previousName,'previousType',previousType,'transitionType',transitionType,'type',type) as unstruct_event_com_snowplowanalytics_mobile_screen_view_1,
-    parse_json('[{"firstEventId":"'||firstEventId||'", "previousSessionId":"'||previousSessionId||'", "sessionId":"'||sessionId||'", "sessionIndex":"'||sessionIndex||'", "userId":"'||userId||'", "eventIndex":"'||eventIndex||'", "storageMechanism":"'||storageMechanism||'", "firstEventTimestamp":"'||firstEventTimestamp||'"}]' ) as contexts_com_snowplowanalytics_snowplow_client_session_1
+    parse_json('[{"firstEventId":"'||firstEventId||'", "previousSessionId":"'||previousSessionId||'", "sessionId":"'||sessionId||'", "sessionIndex":"'||sessionIndex||'", "userId":"'||userId||'", "eventIndex":"'||eventIndex||'", "storageMechanism":"'||storageMechanism||'", "firstEventTimestamp":"'||firstEventTimestamp||'"}]' ) as contexts_com_snowplowanalytics_snowplow_client_session_1,
+    to_variant([OBJECT_CONSTRUCT_KEEP_NULL('latitude', latitude, 'longitude', longitude, 'latitudeLongitudeAccuracy', latitudeLongitudeAccuracy, 'altitude', altitude, 'altitudeAccuracy', altitudeAccuracy, 'bearing', bearing,'speed', speed,'timestamp', timestamp)]) as contexts_com_snowplowanalytics_snowplow_geolocation_context_1
```

Comment:
Would `parse_json` not work here? Or otherwise should we change the other contexts to use `to_variant`? Or is that not needed because the `client_session` is never NULL?